package.json

```diff
@@ -30,14 +30,15 @@
     "debug": "^4.1.1",
     "dotenv": "^8.0.0",
     "express": "^4.17.1",
+    "jsonwebtoken": "^8.5.1",
     "morgan": "^1.9.1",
-    "swagger-ui-express": "^4.0.7",
-    "yamljs": "^0.3.0",
     "pg": "^7.11.0",
     "pg-hstore": "^2.3.3",
     "sequelize": "^5.10.2",
     "sequelize-cli": "^5.5.0",
-    "sqlite3": "^4.0.9"
+    "sqlite3": "^4.0.9",
+    "swagger-ui-express": "^4.0.7",
+    "yamljs": "^0.3.0"
```

Comment:
ditto

Reply:
This is as a result of the husky configuration to format the code before pushing